### PR TITLE
8296130: G1: Remove G1YoungCollector::_target_pause_time_ms

### PR DESCRIPTION
**Pull request by albertnetymk**
Simple change of narrowing down the scope of target-pause-time.

Test: hotspot_gc

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8296130](https://bugs.openjdk.org/browse/JDK-8296130): G1: Remove G1YoungCollector::_target_pause_time_ms

### Reviewers
 * [Ivan Walulya](https://openjdk.org/census#iwalulya) (@walulyai - **Reviewer**) ⚠️ Review applies to [12f4ceb3](https://git.openjdk.org/jdk/pull/10919/files/12f4ceb35042b456b2778e46e901eac2174b1552)
 * [Kim Barrett](https://openjdk.org/census#kbarrett) (@kimbarrett - **Reviewer**) ⚠️ Review applies to [12f4ceb3](https://git.openjdk.org/jdk/pull/10919/files/12f4ceb35042b456b2778e46e901eac2174b1552)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10919/head:pull/10919` \
`$ git checkout pull/10919`

Update a local copy of the PR: \
`$ git checkout pull/10919` \
`$ git pull https://git.openjdk.org/jdk pull/10919/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10919`

View PR using the GUI difftool: \
`$ git pr show -t 10919`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10919.diff">https://git.openjdk.org/jdk/pull/10919.diff</a>

</details>

### CODE REVIEW

**Review by walulyai — APPROVED** [review]

File: src/hotspot/share/gc/g1/g1CollectedHeap.cpp [hunk 1]
```diff
@@ -2024,8 +2024,7 @@ bool G1CollectedHeap::try_collect_concurrently(GCCause::Cause cause,
     // start a concurrent cycle.
     LOG_COLLECT_CONCURRENTLY(cause, "attempt %u", i);
     VM_G1TryInitiateConcMark op(gc_counter,
-                                cause,
-                                policy()->max_pause_time_ms());
+                                cause);
```
Comment (kimbarrett) [inline src/hotspot/share/gc/g1/g1CollectedHeap.cpp:2027]: Consider removing the line break here.

File: src/hotspot/share/gc/g1/g1CollectedHeap.cpp [hunk 2]
```diff
@@ -2652,8 +2649,7 @@ HeapWord* G1CollectedHeap::do_collection_pause(size_t word_size,
   assert_heap_not_locked_and_not_at_safepoint();
   VM_G1CollectForAllocation op(word_size,
                                gc_count_before,
-                               gc_cause,
-                               policy()->max_pause_time_ms());
+                               gc_cause);
```
Comment (kimbarrett) [inline src/hotspot/share/gc/g1/g1CollectedHeap.cpp:2652]: Consider removing the line breaks here.

**Review by kimbarrett — APPROVED** [review]
Looks good.  Just a couple potential formatting suggestions.

**Comment by albertnetymk** [comment]
Thanks for the review.

/integrate

**Pull request closed without merging**